source/mir/random/flex/internal/area.d

```diff
@@ -193,12 +202,18 @@ body
     {
         if (fabs(z) < constants!S.smallExp)
         {
-            area = exp(sh.a) * (ivLength) * (1 + z * S(0.5) + (z * z) * one_div_6
-                                               + (z * z * z) * one_div_24);
+            import std.math : std_exp = exp;
```

Comment:
I will try once again, now that all tests are passing, but last time it did make the difference between passing and failing.